lib/src/pubspec.dart

```diff
@@ -69,6 +70,10 @@ class Pubspec extends PubspecBase {
   // initialization can throw a [PubspecException], that error should also be
   // exposed through [allErrors].
 
+  /// The location of the overrides file, if this pubspec file is incorporating
+  /// overrides from an overrides file.
+  final Uri? overridesLocation;
```

Comment:
I think the location of the overrides file should only belong in entrypoint.dart

Reply:
Done